src/components/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react';
+import { useDispatch } from '../CustomHooks/useDispatch';
+import { useGlobalState } from '../CustomHooks/useGlobalState';
+import { Actions } from '../types/Actions';
+import { filteredTodos } from '../utils/filteredTodos';
+import { FooterButton } from './FooterButton';
+import { SelectedState } from '../types/SelectedState';
+import classNames from 'classnames';
+
+export const Footer = () => {
+  const [selected, setSelected] = useState<SelectedState>({
```

Comment:
also, `selected` is not a very informative variable name

status/filter would be more descriptive